src/main/java/gitflow/actions/RepoActions.java

```diff
@@ -144,11 +143,16 @@
                 actionList.add(new FinishHotfixAction(myRepo));
 
                 //can't publish hotfix if it's already published
-                if (branchUtil.isCurrentBranchPublished() == false) {
+                if (!branchUtil.isCurrentBranchPublished()) {
                     actionList.add(new PublishHotfixAction(myRepo));
                 }
             }
 
+            //RE-INIT ACTIONS
```

Comment:
In most of the use cases people will not need the 're-init' action, and always including it kind of clutters up the interface for daily use.
I had a thought that maybe we can only include these actions if the popup is opened using a modifier key. (IE shift-click the gitflow status will open the menu with these actions. Otherwise the actions won't appear)

Would you mind looking into how to do that? That would be super helpful for other "advanced" features as well. Thanks!

Reply:
Since it's about decluttering the interface a bit, would it be an option to add the options of the plugin to the 'VCS -> Git' menu (if that's possible)?
If we manage that, we could choose to only add such advanced options as this to the menu but not to the `GitflowPopupGroup` for example. From a users perspective I think that's clearer than having to use a modifier key when opening the popup.
Also, I've been looking through the plugin-documentation a bit and I didn't really find anything helpful regarding checking whether a certain key has been pressed or not when opening a pop-up, so this might be easier to do as well.

Also, maybe you could just go ahead and publish a new version of the plugin, compatible with IntelliJ 2018.3? I've been running it locally to test my changes, also using 'feature' and 'hotfix'-features of the plugin, and they work on my machine. That way, users don't have to wait too long for a new version.
I think even without this re-init option a new version would be a useful upgrade, especially if you could also change the supported versions.

Reply:
I hope I got it rigt and to your liking, but please tell me if you need any more changes done :-)